pkg/identity/numericidentity.go

```diff
@@ -55,8 +59,115 @@ const (
 	// ReservedIdentityInit is the identity given to endpoints that have not
 	// received any labels yet.
 	ReservedIdentityInit
+
+	// --------------------------------------------------------------
+	// Special identities for well known cluster components
+
+	// ReservedETCDOperator is the reserved identity used for the etcd-operator
+	// managed by Cilium.
+	ReservedETCDOperator NumericIdentity = 100
+
+	// ReservedCiliumKVStore is the reserved identity used for the kvstore
+	// managed by Cilium (etcd-operator).
+	ReservedCiliumKVStore NumericIdentity = 101
+
+	// ReservedKubeDNS is the reserved identity used for kube-dns.
+	ReservedKubeDNS NumericIdentity = 102
+
+	// ReservedEKSKubeDNS is the reserved identity used for kube-dns on EKS
+	ReservedEKSKubeDNS NumericIdentity = 103
```

Comment:
What is the need for a different identity for kube-dns on EKS? Can you elaborate as to why this is necessary?

EDIT: never mind, the labels are different below. I don't see a better way of doing this, but we are making things specific to implementations that are outside of our control (say Amazon decides to change kube-dns labels). It might be a good idea to mention the exactness of these labels being required to users in the EKS guide?

Reply:
The user does not really have control over this but it definitely makes sense to document it 👍